### PR TITLE
Fix for issue #6178

### PR DESCRIPTION
**Pull request by joneskj55**
This PR addresses issue #6178 by updating the version number in the String.json file as addressed in the issue.

### CODE REVIEW

File: javascript/builtins/String.json [hunk 1]
```diff
@@ -1794,7 +1794,7 @@
                 "version_added": "1.0"
               },
               "webview_android": {
-                "version_added": "1"
+                "version_added": "41"
```
Comment (saschanaz) [inline javascript/builtins/String.json:1797]: This modifies `.replace()` instead of `.repeat()`. Line 1745 should be fixed instead.

Comment (joneskj55) [reply javascript/builtins/String.json]: @saschanaz Thanks! I reverted the previous change and added another commit to this branch with the line 1745 change.

**Review by Elchi3 — APPROVED** [review]
Looks good to me. Thanks @joneskj55 and welcome to mdn-browser-compat-data! :tada:

**Comment by joneskj55** [comment]
Thanks @Elchi3 !

**Pull request merged**